modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/utils/ServiceVariablesJsonSchemaValidator.java

```diff
@@ -71,4 +73,42 @@ public void validateDeployVariables(List<DeployVariable> deployVariables,
             throw new VariableInvalidException(errors);
         }
     }
+
+    /**
+     * Check validation of service configuration map by list of ServiceConfigurationParameter in
+     * registered service.
+     *
+     * @param configurationParameters list of serviceConfigurationParameter in registered service.
+     * @param configurations          service configuration map
+     */
+    public void validateServiceConfiguration(
```

Comment:
Please move this to a different class - ServiceConfigurationVariablesJsonSchemaValidator

and rename the old class as ServiceDeployVariablesJsonSchemaValidator

same applies to ServiceVariablesJsonSchemaGenerator class as well.